src/server.ts

```diff
@@ -27,11 +32,15 @@ app.use(requestLogger);
 
 // Routes
 app.use('/health-check', healthCheckRouter);
-app.use('/users', userRouter);
+app.use('/report', ReportRouter);
+
+app.use('/uploads', express.static(path.join(__dirname, '../' + env.UPLOADS_DIR)));
```

Comment:
Can we only support this in NODE_ENV=development or smth? In production we serve this via nginx either way.